Cargo.toml

```diff
@@ -6,11 +6,17 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-actix-web = "*"
-serde = "*"
-actix-rt = "*"
-serde_derive = "*"
-serde_json = "*"
-shells = "*"
-reqwest = { version = "*", features = ["native-tls", "json", "blocking"] }
-tokio = { version = "*", features = ["full"] }
+actix-web = "4"
```

Comment:
For binaries like the spawn server, I prefer to have all on "*" and check in the Cargo.lock (which I seem to have forgotten here)